src/main/java/com/liferay/faces/util/application/internal/ResourceValidatorImpl.java

```diff
@@ -51,11 +51,7 @@
 	private static final String[] BANNED_SEQUENCES = new String[] {
 			"//", "\\\\", "/\\", "\\/", "..", "./", ".\\", "%"
 		};
-	protected static final String DEFAULT_FACELETS_SUFFIX = ViewHandler.DEFAULT_FACELETS_SUFFIX;
```

Comment:
@stiemannkj1 FWIW, as I recall these constants were defined here like this in order to minimize diffs across branches. `ViewHandler.DEFAULT_FACELETS_SUFFIX` was probably first introduced in JSF 2.0 and would not be present in the util-1.x branch. But since backports have stopped I suppose this is fine.